tests/aiohttp/tox.ini

```diff
@@ -35,5 +35,6 @@ commands =
     docker-compose up -d flask_app_c
     pytest -rPx test_aiohttp_4.py
     docker-compose stop
+    docker-compose down --rmi all
```

Comment:
Do we still need the previous step?

Reply:
I think that can be remove  but let me run some local test to validate.